server/knex/seeds/animation_panels.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
+
+const fs = require('fs');
+const path = require('path');
+const promisify = require('bluebird').promisify;
+const readFileAsync = promisify(fs.readFile);
+
+const exampleDirs = [
+    'recorded_animation_area_bar_charts',
+    'recorded_animation_svg_line_pie_charts',
+    'live_animation_server_monitor'
+].map(
+    dirName => path.join('..', '..', 'examples', 'templates', dirName)
```

Comment:
The paths didn't work for me, I needed to change them like this:
```suggestion
    dirName => path.join(__dirname, '..', '..', '..', 'examples', 'templates', dirName)
```